livre/eq/render/RayCastRenderer.cpp

```diff
@@ -430,9 +429,13 @@ struct RayCastRenderer::Impl
 
     void onFrameRender( const NodeIds& bricks )
     {
+        createAndFillVertexBuffer( bricks );
         size_t index = 0;
         for( const NodeId& brick: bricks )
             renderBrick( brick, index++ );
+
+        glDeleteBuffers( 1, &_posVBO );
+        glFlush();
```

Comment:
In the beginning of onFrameRender() in the createAndFillVertexBuffer() function.